src/ape/api/projects.py

```diff
@@ -137,7 +137,7 @@ def _create_source_dict(
                     hash=compute_checksum(source_path.read_bytes()),
                 ),
                 urls=[],
-                content=source_path.read_text(),
+                content=source_path.read_text("utf8"),
```

Comment:
I'm guessing we will want to do a similar treatment to the solidity plugin? 

https://github.com/ApeWorX/ape-solidity/blob/3dee61309d10b796da6027e9abf4828763fef197/ape_solidity/compiler.py#L29

Reply:
I believe this is at a layer above the compilers?